TYPO3.Media/Classes/TYPO3/Media/Controller/AssetController.php

```diff
@@ -392,8 +401,13 @@ public function addAssetToCollectionAction(Asset $asset, AssetCollection $assetC
      */
     public function deleteAction(Asset $asset)
     {
-        $this->assetRepository->remove($asset);
-        $this->addFlashMessage(sprintf('Asset "%s" has been deleted.', htmlspecialchars($asset->getLabel())));
+        try {
+            $this->assetRepository->remove($asset);
+            $this->addFlashMessage(sprintf('Asset "%s" has been deleted.', htmlspecialchars($asset->getLabel())));
```

Comment:
Yes, will take care of that in https://github.com/neos/neos-development-collection/pull/464/ but it's just a move of a line and not really related to the PR. So if you're fine with me fixing it in the other PR I'd love to see a thumb ;)